templates/conf.d/nginx.conf.erb

```diff
@@ -40,11 +40,16 @@ events {
   <%- end -%>
   worker_connections <%= @worker_connections -%>;
   <%- if @multi_accept == 'on' -%>
-    multi_accept on;
+  multi_accept on;
   <%- end -%>
   <%- if @events_use -%>
-    use <%= @events_use %>;
+  use <%= @events_use %>;
   <%- end -%>
+  <% if @debug_connection && Array(@debug_connection).size > 0 -%>
+    <%- Array(@debug_connection).each do |address| -%>
```

Comment:
The data type is enforced so no need to cast to `Array()`. It's also cleaner to use empty though it's entirely redundant since it'll just loop 0 times
```suggestion
  <% if @debug_connection -%>
    <%- @debug_connection.each do |address| -%>
```

Reply:
Thought to keep Array or Sting, but let's stick with array and cleaner code.
Done.